charts/osm/crds/meshconfig.yaml

```diff
@@ -73,6 +73,17 @@ spec:
                       description: Image for the init container
                       type: string
                       default: "openservicemesh/init:v0.8.3"
+                    resources:
```

Comment:
What would be the use case of this, further would this be configurable at the time of install ?

Reply:
Also would there be any defaults, if its not specified ?

Reply:
I didn't do that to be compliant and be able to reuse corev1's `resourceRequirements`. Consider cpu and memory are not the only possible values here (ie. hugepage limits are also available).

The internal client implementation of corev1's `resourceRequirements` for `limits` and `requests` are actually `map[string]limit`, and the way to support this is the one I used (map type doesn't exist in crd).